be/src/controllers/rest.router.ts

```diff
@@ -60,7 +60,20 @@ router.post('/vocab', (req: Request, res: Response, next: NextFunction) => {
 });
 
 router.put('/vocab/:id', (req: Request, res: Response, next: NextFunction) => {
-  next(new KnowledgeError(501, 'Not Implemented', 'PUT /vocab/:id is not implemented'));
+  const headerName = 'If-Unmodified-Since';
+  const ifUnmodifiedSince: string = req.header(headerName);
```

Comment:
Since we will need this pattern at several places, please move it into a separate function (including basic error handling like _missing_ header and _invalid/unparsable date_)

Reply:
> Since we will need this pattern at several places, please move it into a separate function (including basic error handling like _missing_ header and _invalid/unparsable date_)

Talked and implemented with @dan0bar on his branch, I will wait until he has merged and then use it in this branch.